### PR TITLE
Update info for 2022

### LINKED ISSUE #189: Updating info to 2022
Replace date: Friday, May 20, 2022 
Add: Tulsa Central Library

Remove (Hide?) all visible sponsors
Remove (Hide) Schedule, Replay and Feedback Links

Add: _Sponsor Us_ link on left (form is in this issue)
Add: _Speak_ link on left https://www.papercall.io/200ok2022
Add: _Register_ link on left https://techlahoma.app.neoncrm.com/eventReg.jsp?event=525&

[2022 Sponsorship Prospectus - 200OK (1) (1).pdf](https://github.com/techlahoma/200ok-site/files/7900514/2022.Sponsorship.Prospectus.-.200OK.1.1.pdf)

### PR DESCRIPTION
**Pull request by travismiller**
Closes #189 

Review link: https://deploy-preview-196--200ok.netlify.app/

### CODE REVIEW

File: content/_includes/header.njk [hunk 1]
```diff
@@ -6,28 +6,31 @@
...
-            </a>
+            <a class="button primary ; w-48" href="/sponsorship">Sponsor Us</a>
           </li>
           <li>
-            <a class="button secondary ; w-48 mt-2" href="#schedule" onclick="return scrollToId('schedule')">Schedule</a>
+            <a class="button primary ; w-48 mt-2" href="https://www.papercall.io/200ok2022">Speak</a>
           </li>
           <li>
-            <a class="button secondary ; w-48 mt-2" href="#sponsors" onclick="return scrollToId('sponsors')">Sponsors</a>
+            <a class="button primary ; w-48 mt-2" href="/tickets">Register</a>
```
Comment (maxxcrawford) [inline content/_includes/header.njk:27]:
```suggestion
            <a class="button primary ; w-48 mt-2" href="/tickets" target="_blank">Register</a>
```

File: content/_includes/header.njk [hunk 2]
```diff
@@ -6,28 +6,31 @@
...
           <li>
-            <a class="button primary ; w-48" href="/watch" target="_blank">
-              <span><i class="ri-tv-line ri-2x"></i></span>
-              <span class="whitespace-nowrap">Watch Replay</span>
-            </a>
+            <a class="button primary ; w-48" href="/sponsorship">Sponsor Us</a>
           </li>
           <li>
-            <a class="button secondary ; w-48 mt-2" href="#schedule" onclick="return scrollToId('schedule')">Schedule</a>
+            <a class="button primary ; w-48 mt-2" href="https://www.papercall.io/200ok2022">Speak</a>
```
Comment (maxxcrawford) [inline content/_includes/header.njk:24]:
```suggestion
            <a class="button primary ; w-48 mt-2" href="https://www.papercall.io/200ok2022" target="_blank">Speak</a>
```

**Review by maxxcrawford — APPROVED** [review]
Looks good to me - I have a few nit suggestions and one open question: Is this rendering correctly? We may want to revisit the design of the "taken" spots at some point.

image

**Comment by travismiller** [comment]
Thanks @maxxcrawford!

Suggestions applied. And I've gone ahead and modified the already taken style a bit. We can still revisit later.

image

**Pull request merged**